game/screens/credits.py

```
@@ -1,34 +1,273 @@
+        text_surf = self._fm.render_text(text, self._KEY_TEXT, font_size, font_name=SANS_SERIF_PREFERRED)
+        self.screen.blit(text_surf, text_surf.get_rect(center=face_rect.center))
+
+    def _draw_title_keys(self, text, center_x, top_y,
+                         font_size=34, pad_x=10, pad_y=8, gap=4):
+        """Draw each character of *text* as its own keycap, laid out in a row."""
+        widths = [self._keycap_size(ch, font_size, pad_x, pad_y)[0] for ch in text]
+        _, face_h = self._keycap_size("X", font_size, pad_x, pad_y)
+        total_w = sum(widths) + gap * (len(text) - 1)
+        x = center_x - total_w // 2
+        center_y = top_y + (face_h + self._KEY_DEPTH) // 2
+        for ch, w in zip(text, widths):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)